### PR TITLE
Resultpage

### PR DESCRIPTION
**Pull request by Rockyraven**
created result page
final score card added
question result list updated

### CODE REVIEW

File: src/context/questionContext.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ const questionContext = createContext();
 const QuestionProvider = ({children}) => {
     const [ questionCategoryList, setQuestionCategoryList ] = useState([]);
     const [ resultInfo, setResultInfo ] = useState([]);
+  const [score, setScore] = useState(0);
+
```
Comment (Shubhamkr2610) [inline src/context/questionContext.js:12]: Suggestion: you can use useReducer here to manage state

**Review by Shubhamkr2610 — APPROVED** [review]
Else everything looks good to me , you can go for merge

**Review by arulk8 — APPROVED** [review]
good to go

File: src/context/questionContext.js (hunk 1 above)
Comment (Rockyraven) [reply src/context/questionContext.js:12]: ok I will do it later

**Pull request merged**